mriqc/qc/functional.py

```diff
@@ -21,32 +21,73 @@
 - **Foreground-Background energy ratio** (:py:func:`~mriqc.qc.anatomical.fber`,  [Shehzad2015]_).
 - **Full-width half maximum smoothness** (``fwhm_*``).
 - **Signal-to-noise ratio** (:py:func:`~mriqc.qc.anatomical.snr`).
-- **Summary statistics** (``summary_*_*``).
+- **Summary statistics** (:py:func:`~mriqc.qc.anatomical.summary_stats`).
 
 
 Measures for the temporal information
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
 - **DVARS** - D referring to temporal derivative of timecourses, VARS referring to
-  RMS variance over voxels (``dvars``), calculated
+  RMS variance over voxels ([Power2012]_ ``dvars``) indexes the rate of change of
+  BOLD signal across the entire brain at each frame of data. DVARS is calculated
   `with nipype <http://nipype.readthedocs.io/en/latest/interfaces/generated/\
-nipype.algorithms.confounds.html#computedvars>`_ before motion correction.
-- **Ghost to Signal Ratio** (:py:func:`~mriqc.qc.functional.gsr`, ``ghost_*``:
-  along the two possible phase-encoding axes **x**, **y**.
-- **Global Correlation** (:py:func:`~mriqc.qc.functional.gcor`, ``gcor``).
-- **Temporal SNR** (:abbr:`tSNR (temporal SNR)`, ``tsnr``) is the median value
+nipype.algorithms.confounds.html#computedvars>`_ before motion correction:
```

Comment:
It would be also good to explain or add reference to the standardized versions of DVARS.